pkg/config/config.go

```diff
@@ -589,6 +589,8 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("leader_election", false)
 	config.BindEnvAndSetDefault("kube_resources_namespace", "")
 	config.BindEnvAndSetDefault("kube_cache_sync_timeout_seconds", 5)
+	config.BindEnvAndSetDefault("kube_cache_sync_total_retry_seconds", 300)
```

Comment:
I think we can avoid exposing this setting as it could conflict with the other two.
if someone already has a `kube_cache_sync_timeout_seconds` > 300 how will it behave? will it at least try one time?

Reply:
Sure, I can remove this setting.  if` kube_cache_sync_timeout_seconds `> 300, It will at least try one time.
https://github.com/DataDog/datadog-agent/blob/6d1af9ab6a0500dce232f1fc0fb7102f266f199f/pkg/util/kubernetes/apiserver/util.go#L86-L91